src/main.js

```diff
@@ -24,12 +26,13 @@ k.scene('forest_junction', (enter_tag) =>
 k.scene('campus_house_1', (enter_tag) =>
     bootstrap(campusHouse1, { enter_tag })
 );
+k.scene('fishing', (enter_tag) => miniGameBootstrap(fishing, { enter_tag }));
 
 // Game Machine Scenes
 k.scene('startScreen', gameStartScreen);
 k.scene('lose', loseScreen);
 
-k.go('start');
+k.go('fishing');
```

Comment:
Just revert this back to start and we are good to go 🙌🏻 @fullspeccoder

Reply:
Done! Thank you so much for the thoughts and critique on the game 😌 I'm glad it'll finally be in the game!